examples/pause_soundfile/sketch.js

```diff
@@ -14,16 +15,35 @@ function setup() {
   createCanvas(400, 400);
   background(0, 255, 0);
 
-  soundFile.loop();
+  if (getAudioContext().state === 'running') {
+    soundFile.loop();
+  }
+
   createP('Press any key to pause. Resume when the key is released')
 }
 
+function touchStarted() {
+  if (getAudioContext().state !== 'running') {
+    getAudioContext().resume();
```

Comment:
We have a `userStartAudio` method that can be useful for this, too https://p5js.org/reference/#/p5.sound/userStartAudio

Reply:
I've updated the sketch to use `userStartAudio` instead.